Sources/CircularProgress/util.swift

```diff
@@ -276,6 +276,21 @@ final class ProgressCircleShapeLayer: CAShapeLayer {
 	}
 }
 
+/**
+Shows the indeterminate state, when it's activated.
+
+It draws part of a circle that gets animated into a looping motion around its core.
+*/
+final class IndeterminateShapeLayer: CAShapeLayer {
```

Comment:
I just realized, shouldn't the name be `IndeterminateShapeLayer` => `IndeterminateCircleShapeLayer`?

Reply:
I thought ‘...CircleShapeLayer’ was the convention because of ‘ProgressCircleShapeLayer’. If it’s not, then ‘IndeterminateShapeLayer’ sounds fine to me. 👍